### DISCUSSION TITLE
icd10  with CUI and TUI

### DISCUSSION THREAD

**Discussion by Siddhijain16** [explosion/spaCy#8482] [Help: Coding & Implementations] [answered]
Can we get icd10 value of entity along with CUI ( Concept Unique Identifier) and TUI (semantic type )  ??
We are able to get CUI and TUI with the help of "[scispacy.linking](https://github.com/allenai/scispacy#example-usage-1) " but requires ICD10 also .
Is there any way to map CUI and TUI to get ICD10 ?
Please suggest how can I get ICD10 of an entity ...

**Comment by polm** [comment, marked as answer]
Hello, sorry to hear you're having trouble. scispaCy is a separate project from spaCy and we do not maintain it and cannot help you with it. I recommend you ask at their repo.

https://github.com/allenai/scispacy

Also note that posting screenshots of code is considered unhelpful, and you should copy/paste text or code instead.
